### PR TITLE
i18n(fr) Big Update for `typescript.mdx`

### PR DESCRIPTION
**Pull request by McFlyPartages**
#### Description (required)

Update french translation for typescript.mdx file with PR #2546 #2690 #2765 #2727 #2728 #2891 #2944 #2963 #2948 #2993 #3263 #4059 #4446 #4899 #5051 #5070 #5284 #5547 #4266 #5765 #6109 #6267

### CODE REVIEW

File: src/content/docs/fr/install/manual.mdx [hunk 1]
```diff
@@ -172,7 +172,7 @@ Enfin, créez `src/env.d.ts`` pour que TypeScript sache des types ambiants dispo
 /// <reference types="astro/client" />
 ```
 
-📚 Lisez le [guide d'installation de TypeScript](/fr/guides/typescript/#mise-en-place) d'Astro pour plus informations.
+📚 Lisez le [guide d'installation de TypeScript](/fr/guides/typescript/#configuration) d'Astro pour plus informations.
```
Comment (thomasbnt) [inline src/content/docs/fr/install/manual.mdx:175]:
```suggestion
<ReadMore>Lisez le [guide d'installation de TypeScript](/fr/guides/typescript/#configuration) d'Astro pour plus informations.</ReadMore>
```

Et ne pas oublier d'importer ligne 9 `import ReadMore from '~/components/ReadMore.astro'`. Voir https://github.com/withastro/docs/blob/c931bcf2f1903846cf6d88f8aefc86dc4f58e570/src/content/docs/en/install/manual.mdx

**Review by thomasbnt — CHANGES_REQUESTED** [review]

**Comment by McFlyPartages** [comment]
C'est validé @thomasbnt merci ;)

File: src/content/docs/fr/guides/typescript.mdx [hunk 2]
```diff
@@ -62,22 +118,190 @@ import Layout from '@layouts/Layout.astro';
...
+    "build": "astro build",
+    "build": "astro check && astro build",
+  },
+```
+
+:::note
+`astro check` vérifie tous les fichiers inclus dans votre projet TypeScript. Pour vérifier les types dans les fichiers Svelte et Vue, vous pouvez utiliser les paquets [`svelte-check`](https://www.npmjs.com/package/svelte-check) et [`vue-tsc`](https://www.npmjs.com/package/vue-tsc) respectivement.
+:::
+
+import ReadMore from '~/components/ReadMore.astro'
```
Comment (thomasbnt) [inline src/content/docs/fr/guides/typescript.mdx:285]:
```suggestion
```
https://github.com/withastro/docs/actions/runs/7515324553/job/20459129743?pr=6372

**Review by thomasbnt — CHANGES_REQUESTED** [review]

**Review by thomasbnt — APPROVED** [review]
LGTM!

**Review by dreyfus92 — APPROVED** [review]
LGTM 😁

**Pull request merged**